.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto
```

Comment:
I forgot to include this file when resetting the repo, so line endings got a bit messed up (CRLF is evil), my bad!